src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/layout.tsx

```diff
@@ -53,6 +54,15 @@ export type FixLayoutProps = {
 };
 
 const FixLayout = (props: FixLayoutProps) => {
+  const pathname = usePathname();
+  const [isHighRiskDataBreach, setIsHighRiskDataBreach] = useState(false);
+
+  useEffect(() => {
+    if (pathname.includes("high-risk-data-breaches")) {
+      setIsHighRiskDataBreach(true);
+    }
+  }, [pathname]);
```

Comment:
I don't think any side effects are actually happening here, nor that this variable changes in response to anything else, so I think you should be able to just do:

```suggestion
  const isHighRiskDataBreach = pathname.includes("high-risk-data-breaches");
```